sklearn/linear_model/_coordinate_descent.py

```diff
@@ -1507,6 +1506,12 @@ def __init__(self, *, eps=1e-3, n_alphas=100, alphas=None,
             cv=cv, verbose=verbose, n_jobs=n_jobs, positive=positive,
             random_state=random_state, selection=selection)
 
+    def _get_model(self):
+        return Lasso()
```

Comment:
I feel like returning the class rather than an instance is more natural because we are going to use `set_params` on it later anyway

```suggestion
    def _get_model_class(self):
        return Lasso
```

Reply:
so  `model.set_params(**common_params)` becomes `model = self._get_model_class(**common_params)` ?

Reply:
@NicolasHug Actually we call `model.get_params()`  before we set the parameters, so we need a "blank" instance available before `set_params()`